docs/using_tags.md

```diff
@@ -7,12 +7,12 @@ They work in a similar way as [persistence cache tags in eZ Platform v2](https:/
 - Like an index in a database it is typically used for anything relevant that represent the given cache item
 - Used for cache invalidation
 
-There are also other aspects to this bundle, like the fact that by using Varnish xkey over BAN we can now do soft purge,
-but more on that in [varnish/README.md](varnish/README.md) and the corresponding VCL file.
+It works across all supported proxies _(see ["drivers"](drivers.md))_ by eZ Platform:
+- Symfony Proxy _(PHP based for single server usage, primarily for smaller web sites)_
+- [Varnish](https://varnish-cache.org/) with [xkey module](https://github.com/varnish/varnish-modules) _or_ [Varnish Plus](https://www.varnish-software.com/products/varnish-plus/) _(High performance reverse proxy)_
+- [Fastly](https://www.fastly.com/) _(High performance reverse proxy, orignally based on Varnish, worldwide as a CDN, available in eZ Platform Enterprise)_
```

Comment:
Typo on "originally".

Reply:
Fixed in https://github.com/ezsystems/ezplatform-http-cache/commit/3c7ff728893662e94c10224e26541f6100d96f34